### PR TITLE
[FIX] account_banking_sepa_direct_debit: Fix report mandate

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnatica TT20651
@carlosdauden @Tardo Review...

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: account_banking_sepa_direct_debit/views/report_sepa_direct_debit_mandate.xml [hunk 1]
```diff
@@ -6,10 +6,8 @@
         <t t-set="is_not_english" t-value="doc.partner_id.lang and doc.partner_id.lang[:2] != 'en'" />
         <div class="page sepa_direct_debit">
             <div class="row mt0">
-                <div class="col-3">
-                    <img t-if="doc.company_id.logo" t-att-src="'data:image/png;base64,%s' % doc.company_id.logo" style="max-height: 45px;"/>
-                </div>
                 <div class="col-12 text-center">
+                    <img t-if="doc.company_id.logo" t-attf-src="data:image/png;base64,#{doc.company_id.logo}" style="max-height: 45px;" class="float-left"/>
```
Comment (Tardo) [inline account_banking_sepa_direct_debit/views/report_sepa_direct_debit_mandate.xml]:
```suggestion
                    <img t-if="doc.company_id.logo" t-att-src="image_data_uri(doc.company_id.logo)" style="max-height: 45px;" class="float-left"/>
```

**Review by Tardo — APPROVED** [review]
LGTM. Only a suggested change...

**Comment by sergio-teruel** [comment]
Thanks @Tardo

**Comment by pedrobaeza** [comment]
Please squash both commits

**Comment by sergio-teruel** [comment]
squash Done

**Pull request merged**